packages/whale-api-client/__tests__/api/poolpair.test.ts

```diff
@@ -162,12 +162,36 @@ describe('get', () => {
   })
 
   it('should throw error as numeric string is expected', async () => {
-    await expect(client.poolpair.get('A-B')).rejects.toThrow(WhaleApiException)
-    await expect(client.poolpair.get('A-B')).rejects.toThrow('Validation failed (numeric string is expected)')
+    expect.assertions(2)
+    try {
+      await client.poolpair.get('A-B')
+      throw new Error('should not reach here')
+    } catch (err) {
```

Comment:
```suggestion
    expect.assertions(2)
    try {
      await client.poolpair.get('A-B')
    } catch (err) {
```

With `expect.assertions` you don't need `throw new Error` since it does the same thing. I think using expect.assertions is clean when doing try catch test assertions. Since `throw new Error` is kind of anomalous.